frontend/README.md

```diff
@@ -1,14 +1,5 @@
 # Corona Game
 
-## Development server
-
-### Install
-- `cd frontend`
-- `npm i`
-
-### Run
-- `npm start` for a dev server at `http://localhost:4200/`
-
 ## Code scaffolding
 Run `npx ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
```

Comment:
What about this? Consider removing it as well.

Reply:
Feel free to do it yourself.